tests/publiccloud/bsc_1205002.pm

```diff
@@ -0,0 +1,41 @@
+# SUSE's openQA tests
+#
+# Copyright 2023 SUSE LLC
+# SPDX-License-Identifier: FSFAP
+
+# Summary: Test to check for bsc#1205002
+#
+# Maintainer: <qa-c@suse.de>
+
+use base "publiccloud::basetest";
+use strict;
+use warnings;
+use testapi;
+use utils;
+use publiccloud::ec2;
+use publiccloud::ssh_interactive "select_host_console";
+
+sub run {
+    my ($self, $args) = @_;
+    select_host_console();    # select console on the host, not the PC instance
+
+    my $provider = $self->provider_factory();
+    $args->{my_provider} = $provider;
+
+    $args->{my_instance}->ssh_assert_script_run("sudo grub2-mkconfig -o /boot/grub2/grub.cfg");
+
+    $provider->stop_instance($args->{my_instance});
+
+    my $instance_type = get_var('PUBLIC_CLOUD_NEW_INSTANCE_TYPE', 't2.large');
+    $provider->change_instance_type($instance_type);
+
+    $provider->start_instance($args->{my_instance});
+
+    $args->{my_instance}->ssh_assert_script_run("echo we can login");
```

Comment:
what if we will run smoketest after that ? e.g. https://openqa.suse.de/tests/11523127/modules/smoketest/steps/1/src

Reply:
Yes.  Makes sense.